### DISCUSSION TITLE
Ceiling Fan

### DISCUSSION THREAD

**Discussion by welchap70** [itavero/homebridge-z2m#653] [Q&A]
Hi  - just setup Zigbee2Mqtt and Homebridge and I have a ceiling fan switch (https://www.zigbee2mqtt.io/devices/SSWF01G.html#mercator-sswf01g)  - I can see all the fan features in the zigbee2mqtt logs eg : 'zigbee2mqtt/Bedroom Fan', payload '{"fan_mode":"off","fan_state":"OFF","linkquality":255,"state":"OFF"}' but I cannot see this in Homebrige - all I can see is the light switch that is exposed.  How do I configure homebridge-z2m to expose the fan controls (on off, fan speed)? 
Thanks
Andy

**Comment by itavero** [comment]
This plugin does currently not support Fan type devices. Also see #333
